### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by muhammedkoca003** [adonisjs/docs#379]
Hi guys,

I think you made a typo in

https://adonisjs.com/docs/4.1/query-builder

```
innerJoin
await Database
  .table('users')
  .innerJoin('accounts', 'user.id', 'accounts.user_id')
```

shouldn't this be:
.innerJoin('accounts', **'users.id'** , 'accounts.user_id') ?

**Comment by fians** [comment]
@muhammedkoca003 Just want to let you know that it has been fixed here #391, however it seems the changes has not reflected on the site yet.

**Comment by RomainLanz** [comment]
Closing!
